docs/global/modules/core/pages/howto/modelql.adoc

```diff
@@ -44,6 +44,24 @@ val result: List<StaticMethodDeclaration> = client.query { root ->
 }
 --
 
+=== Writing with the type-safe ModelQL API
```

Comment:
This could easily be a separate how-to guide.

Reply:
What about this thread?

Reply:
It's done. I moved it to a separate file/guide.

Reply:
Ah, didn't see that. Should we try to follow the naming principles proposed here: https://diataxis.fr/how-to-guides/#pay-attention-to-naming